src/util.ts

```diff
@@ -168,6 +168,30 @@ export function formatAsUTCISO(
   return resultString;
 }
 
+/**
+ * Examines the runtime environment and returns the appropriate tracking string.
+ * @returns {string} metrics tracking string based on the current runtime environment.
+ */
+export function getRuntimeTrackingString(): string {
+  if (
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    globalThis.Deno &&
```

Comment:
Should we add Deno types? https://deno.land/manual@v1.35.1/advanced/typescript/configuration#targeting-deno-and-the-browser